frontend/pages/hosts/details/MacSettingsModal/MacSettingsTable/MacSettingsTableConfig.tsx

```diff
@@ -148,4 +138,22 @@ export const generateTableData = (
   return rows;
```

Comment:
The bug arose because previously `rows` was initialized as empty array. Now it is initialized to profiles.